libmscore/layout.cpp

```diff
@@ -2319,6 +2320,7 @@ bool Score::layoutSystem(qreal& minWidth, qreal systemWidth, bool isFirstSystem,
             if (continueFlag
                || pbreak
                || (nt == Element::Type::VBOX || nt == Element::Type::TBOX || nt == Element::Type::FBOX)
+               || (ct == Element::Type::VBOX || ct == Element::Type::TBOX || ct == Element::Type::FBOX)
```

Comment:
While this makes sense on the surface, I do think there needs to be a *lot* of testing of this change.  Perhaps there was some reason why we weren't breaking the loop right away - something we need to calculate or update later in the loop or something - that will stop working.  I don't have anything specific in mind, but I just kmow how fragile this code has proven in the past.